contents/blog/inflated-risk-seems-riskier.md

```diff
@@ -0,0 +1,78 @@
+---
+date: 2021-03-09
+title: How we justified quitting our jobs and financing PostHog early on
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Let me tell you a story of the financial pressures that face new founders and how that pressure motivated Tim and me to make an epic ~~pivot~~ [leap](https://posthog.com/blog/story-about-pivots), from bootstrapping to a VC-backed startup.
+
+I began this journey because I had spent a lot of time in the world of enterprise software sales, and wanted to work on a product that could be adopted bottom up. I felt like it was the future of software, even that used by huge companies. I convinced Tim to join me with a dubious prototype and confidence of a salesman that the product was pretty much done and ready to sell - so simple right! We both saved a year’s living costs and planned to build and launch for 6 months and then to check in if we'd run for a further 6 months. We took the “risky” leap...
+
+<iframe src="https://giphy.com/embed/Ihs8jRSnrj5JeXOyNO" width="480" height="343" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/fallontonight-jimmy-fallon-tonight-show-starring-Ihs8jRSnrj5JeXOyNO">via GIPHY</a></p>
+
+## Quitting your job can be the most sensible thing to do
+
+Risk is different depending on your skills and experience. Take a look at the demand for your services on the open market. Will the market demand hold for a year or two? Do you have experience contracting or freelancing? If not, do your research and experiment so that you can find your true risk profile - it could be much lower risk than you think.
+
+Tim and I both code but he’s the (so much) stronger developer. Thus he went full-time and I negotiated to go part time 2 days a week. That money went straight into the business. It minimized our shared risk and allowed us to get coworking space, pay for tools and ads without drawing on savings for these - given we weren't going to pay ourselves.
+
+Quite simply, we would spend a set amount of our savings and time, and if it didn't work out, we were fortunate enough to know we'd probably be able to get another job. That takes a lot of privilege because of years working in tech, and our lives leading up to that, and we can only imagine how the world could look now if that were the case for more people.
+
+## You’ll have to decide to bootstrap or go for VC
```

Comment:
Somewhere inside here we could link to the James' Golden Handbook to Fundraising https://posthog.com/blog/raising-3m-for-os#why-raise-money-at-all